mupdf-sys/Cargo.toml

```diff
@@ -34,6 +34,19 @@ license = "GPL-3.0"
 links="mupdf-wrapper"
 repository = "https://github.com/messense/mupdf-rs"
 
+[features]
+default = ["noto-small", "no-cjk"]
+noto-small = []
+no-cjk = []
+tofu = []
+no-js = []
```

Comment:
can we make it `js` not `no-js`? seems odd to use feature flags to disable feature rather than enable feature?

Reply:
I agree it's a bit counterintuitive, I was thinking of it this way because the options in config.h are automatically enabled.
So should we disable all of the options and enable them manually?

Reply:
We can enable some features by default, not a big issue I think.